src/neural/writer.h

```diff
@@ -49,21 +49,41 @@ struct V5TrainingData {
   // For input type 3 contains enpassant column as a mask.
   uint8_t side_to_move_or_enpassant;
   uint8_t rule50_count;
-  // For input type 3 contains a bit field indicating the transform that was
-  // used and the original side to move info.
-  // Side to move is in the top bit, transform in the lower bits.
+  // Bitfield with the following allocation:
+  //  bit 7: side to move (input type 3)
+  //  bit 6: position marked for deletion by the rescorer (never set by lc0)
+  //  bit 5: game adjudicated (v6)
+  //  bit 4: max game length exceeded (v6)
+  //  bit 3: best_q is a proof (v6)
```

Comment:
I don't think bit 3 is implemented as I intended.  If best_q is 1.0 and this bit is set, then all good.
If best_q is 0 and this bit is set, we haven't ruled out that moves other than best_q are actually wins.
Similary if its -1 and this bit is set we haven't ruled out draws or wins for the other moves.

Reply:
Should work as intended after the last commits.